cmd/dlv/cmds/commands.go

```diff
@@ -411,6 +412,8 @@ func dapCmd(cmd *cobra.Command, args []string) {
 
 		if headless {
 			fmt.Fprintf(os.Stderr, "Warning: dap mode is always headless\n")
+		} else {
+			headless = true
```

Comment:
how about unconditionally set to headless. (dap mode is always headless)

Reply:
Do you mean change the error message?

Reply:
```
if headless {
   fmt.Fprintf(...)
}
headless = true
```

Removing the error message completely is another thing to consider - currently this doesn't provide a warning for  `dlv dap --headless=false`.

Reply:
I removed the error message and added more info to he help documentation.

The idea was that the default is false, so when people don't specify the flag (knowing that the server is always headless), then are not spammed. Is there a way to tell if false is coming from default or from a value the user specified explicitly?

Reply:
You can use the following to see if the flag has been set explicitly: https://pkg.go.dev/github.com/spf13/pflag?utm_source=godoc#FlagSet.Changed.